### PR TITLE
feat: add ads-b telemetry service

### PR DESCRIPTION
**Pull request by ServiceDog**
Adds the `AdsbClient` and adds support for the `bytes` GRPC type (`BYTEA` in psql)

### CODE REVIEW

**Comment by ServiceDog** [comment]
Currently debugging error on `rust-example-run`:
```
2023-02-10 10:12:56 | ERROR | server/src/postgres/mod.rs:62 — error executing DB query: error serializing parameter 0: cannot convert between the Rust type `i32` and the Postgres type `int8`
```
I believe this is regarding the INT4 type I added, but I can't tell where it's getting converted to int8

Another odd warning/error from the logs:
```
2023-02-10 10:00:47 | INFO  | server/src/postgres/mod.rs:924 — Creating database tables.
2023-02-10 10:00:47 | INFO  | server/src/postgres/mod.rs:255 — composing create table query for [vertiport]
2023-02-10 10:00:47 | INFO  | server/src/postgres/mod.rs:255 — composing create table query for [vertipad]
2023-02-10 10:00:47 | ERROR | server/src/postgres/mod.rs:311 — Failed to create indices for table [flight_plan]: db error: ERROR: there is no unique constraint matching given keys for referenced table vertiport
2023-02-10 10:00:47 | INFO  | server/src/postgres/mod.rs:255 — composing create table query for [vehicle]
2023-02-10 10:00:47 | INFO  | server/src/postgres/mod.rs:255 — composing create table query for [adsb]
2023-02-10 10:00:47 | INFO  | server/src/postgres/mod.rs:255 — composing create table query for [flight_plan]
2023-02-10 10:00:47 | ERROR | server/src/postgres/mod.rs:311 — Failed to create indices for table [flight_plan]: db error: ERROR: there is no unique constraint matching given keys for referenced table vertipad
2023-02-10 10:00:47 | INFO  | server/src/postgres/mod.rs:255 — composing create table query for [itinerary]
2023-02-10 10:00:47 | INFO  | server/src/main.rs:52 — PSQL Rebuild completed.
```
The _get_indices_etc. function for "flight_plan" appears to be called on "vertipad"'s init_table

File: client-grpc/examples/grpc.rs [hunk 1]
```diff
@@ -88,6 +90,152 @@ async fn get_itineraries() -> Result<itinerary::List, Status> {
     }
 }
 
+/// Example AdsbClient
+/// Assuming the server is running, this method calls `client.itineraries` and
```
Comment (owlot) [inline client-grpc/examples/grpc.rs:94]: Incorrect documentation. This method calls the AdsbClient, not itineraries

File: server/src/resources/vertiport/mod.rs [hunk 2]
```diff
@@ -80,6 +80,13 @@ impl Resource for GenericResource<Data> {
             ]),
         }
     }
+
+    fn get_table_indices() -> Vec<String> {
```
Comment (owlot) [inline server/src/resources/vertiport/mod.rs:84]: Not sure why this was added. There's a default implementation of this function present in the Trait itself to return an empty list if no indices are needed (yet).

Comment (ServiceDog) [reply server/src/resources/vertiport/mod.rs]: Removed, was an artifact from trying to debug this error, which I still haven't solved:
```
arrow-svc-storage       | 2023-02-11 18:49:01 | INFO  | server/src/postgres/mod.rs:227 — Initializing global shared psql pool.
arrow-svc-storage       | Initializing database connection pool.
arrow-svc-storage       | 2023-02-11 18:49:01 | INFO  | server/src/postgres/mod.rs:154 — Setting up TLS connection with client cert and key.
arrow-svc-storage       | 2023-02-11 18:49:01 | INFO  | server/src/postgres/mod.rs:197 — Successfully created PostgresPool.
arrow-svc-storage       | 2023-02-11 18:49:01 | INFO  | server/src/postgres/mod.rs:924 — Creating database tables.
arrow-svc-storage       | 2023-02-11 18:49:01 | INFO  | server/src/postgres/mod.rs:255 — composing create table query for [vertiport]
arrow-svc-storage       | 2023-02-11 18:49:01 | INFO  | server/src/postgres/mod.rs:255 — composing create table query for [vertipad]
arrow-svc-storage       | 2023-02-11 18:49:01 | ERROR | server/src/postgres/mod.rs:311 — Failed to create indices for table [flight_plan]: db error: ERROR: there is no unique constraint matching given keys for referenced table vertiport
arrow-svc-storage       | 2023-02-11 18:49:01 | INFO  | server/src/postgres/mod.rs:255 — composing create table query for [vehicle]
arrow-svc-storage       | 2023-02-11 18:49:01 | ERROR | server/src/postgres/mod.rs:311 — Failed to create indices for table [flight_plan]: db error: ERROR: there is no unique constraint matching given keys for referenced table vertiport
arrow-svc-storage       | 2023-02-11 18:49:01 | INFO  | server/src/postgres/mod.rs:255 — composing create table query for [adsb]
arrow-svc-storage       | 2023-02-11 18:49:01 | INFO  | server/src/postgres/mod.rs:255 — composing create table query for [flight_plan]
arrow-svc-storage       | 2023-02-11 18:49:01 | ERROR | server/src/postgres/mod.rs:311 — Failed to create indices for table [flight_plan]: db error: ERROR: there is no unique constraint matching given keys for referenced table vertipad
arrow-svc-storage       | 2023-02-11 18:49:01 | INFO  | server/src/postgres/mod.rs:255 — composing create table query for [itinerary]
arrow-svc-storage       | 2023-02-11 18:49:01 | INFO  | server/src/postgres/mod.rs:227 — Initializing global shared psql pool.
```

File: client-grpc/examples/grpc.rs (hunk 1 above)
Comment (ServiceDog) [reply client-grpc/examples/grpc.rs]: Fixed with https://github.com/Arrow-air/svc-storage/pull/45/commits/b5715a1403023460814b2f94c7a84306ab603df2

**Review by owlot — APPROVED** [review]

**Pull request merged**